addons/mail/security/ir.model.access.csv

```diff
@@ -17,7 +17,10 @@ access_mail_channel_all,mail.group.all,model_mail_channel,,1,0,0,0
 access_mail_channel_user,mail.group.user,model_mail_channel,base.group_user,1,1,1,1
 access_mail_channel_partner_public,mail.channel.partner.public,model_mail_channel_partner,base.group_public,1,0,0,0
 access_mail_channel_partner_portal,mail.channel.partner.portal,model_mail_channel_partner,base.group_portal,1,1,1,1
-access mail_channel_partner_user,mail.channel.partner.user,model_mail_channel_partner,base.group_user,1,1,1,1
+access_mail_channel_partner_user,mail.channel.partner.user,model_mail_channel_partner,base.group_user,1,1,1,1
+access_mail_moderation_email_portal,mail.mail.moderation.email.portal,model_mail_moderation_email,base.group_portal,1,0,0,0
+access_mail_moderation_email_user,mail.mail.moderation.email.user,model_mail_moderation_email,base.group_user,1,1,1,1

```

Comment:
I think (but I am not sure) that actually no-one except administrator should have access to ban data. Channel model should delegate this right. I suggest to talk about this a bit later, once all other remarks are taken into account.